src/main/java/Katleen.java

```diff
@@ -0,0 +1,65 @@
+import java.util.Scanner;
+
+public class Katleen {
+    public static final String LINE = "____________________________________________________________";
+
+    public static void main(String[] args) {
+        System.out.println(LINE);
+        System.out.println("Hello! I'm Katleen.");
+        System.out.println("What can I do for you?");
+        System.out.println(LINE);
+
+        Task[] tasks = new Task[100];
+
+        Scanner in = new Scanner(System.in);
+        String text = in.nextLine();
+        while (!text.equals("bye")) {
+            String[] textSplit = text.split(" ");
+            String cmdWord = textSplit[0];
+            System.out.println("Cmdword is: " + cmdWord);
+            System.out.println(LINE);
+            if (cmdWord.equals("mark")) {
```

Comment:
Do you think maybe using a switch statement is better than so many if else statements? I think it would make the code more readable.